### DISCUSSION TITLE
[Feature Request] Add "Send Page To Device" to context menu

### DISCUSSION THREAD

**Discussion by haoadoreorange** [piroor/treestyletab#3116] [General]
I know there's "Send Tab/Tree To Device" that TST offer and its limitation. I never use the "Send Tree" anyway.

Is it possible at all to add the "Send Page To Device" native button of Firefox to the context menu in side bar ? I use `userChrome.css` to hide the top tabbar, so everytime I wanna use the functionality I need to focus to the tab and click directly on the page to have the option. It would be nice if I can do that directly on the sidebar.

**Comment by piroor** [comment]
There was a bug about making the feature callable via WebExtensions API ([1417183 - Provide a web extensions API-based way to send pages between devices](https://bugzilla.mozilla.org/show_bug.cgi?id=1417183)) but it was closed as "WONTFIX", so sadly TST can't do anything about this.

userChrome.js may be a solution to do this bypassing WebExtensions API, but it looks to be beyond the scope of this project. I move this to discussions from issues and hoping someone add further comments.

**Comment by haoadoreorange** [comment]
Do you have a clue as to how to bypass it using userChrome.js ?
